.changeset/sharp-rockets-do.md

```diff
@@ -0,0 +1,6 @@
+---
+"@itwin/presentation-components": minor
+---
+
+- Bump itwinui to 3.\*.
+- Edit `UseFilterablePresentationTree` API by removing `containerRef`.
```

Comment:
User friendly "bump" message. The other, IMO, is not necessary as it's basically just reacting to the bump.

```suggestion
Bump iTwinUI package dependencies to 3.x.
```